.travis.yml

```diff
@@ -1,24 +1,28 @@
 language: node_js
 node_js:
-- iojs
+  - iojs
 before_script:
-- if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then curl https://gist.githubusercontent.com/santiycr/5139565/raw/sauce_connect_setup.sh
-  | bash; fi
+  - if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then curl https://gist.githubusercontent.com/santiycr/5139565/raw/sauce_connect_setup.sh | bash; fi
 before_install:
-- export CHROME_BIN=chromium-browser
-- export DISPLAY=:99.0
-- sh -e /etc/init.d/xvfb start
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+script: grunt test && grunt coveralls
```

Comment:
`npm test` can receive a arguments in npm 2.0 and up. So, we can do `npm test -- --coverage` and then the script can see whether that argument was available.
Alternatively, the value for `npm test` could be something like `grunt test && if [ "$TRAVIS" = "true" ] grunt coveralls]` (or however you check for env variables.